lib/web/apiserver.go

```diff
@@ -357,6 +357,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	// MFA private endpoints.
 	h.GET("/webapi/mfa/devices", h.WithAuth(h.getMFADevicesHandle))
 	h.POST("/webapi/mfa/authenticatechallenge", h.WithAuth(h.createAuthenticateChallengeHandle))
+	h.POST("/webapi/mfa/token/:token/totpsecrets", h.WithAuth(h.createTOTPSecretsHandle))
```

Comment:
Could we be more generic and allow the creation of MFARegisterChallenge, instead of catering just to OTP? This will come in handy for other MFA forms as well.

Reply:
deleted (handled with CreateRegisterChallenge)